Fix failing BasicStandardEndpointMapper tests

Some tests are failing in the CI job and locally:

BasicStandardEndpointMapperTest#testAlwaysServeStaticContent

`
java.io.IOException: Server returned HTTP response code: 500 for URL: http://localhost:1425/seconda.html
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1997)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1589)
	at java.base/java.net.URL.openStream(URL.java:1161)
	at org.apache.commons.io.IOUtils.toString(IOUtils.java:2953)
	at org.apache.commons.io.IOUtils.toString(IOUtils.java:2910)
	at org.apache.commons.io.IOUtils.toString(IOUtils.java:2926)
	at org.carapaceproxy.server.mapper.BasicStandardEndpointMapperTest.testAlwaysServeStaticContent(BasicStandardEndpointMapperTest.java:414)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.github.tomakehurst.wiremock.junit.WireMockRule$1.evaluate(WireMockRule.java:79)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
`

and BasicStandardEndpointMapperTest#testCustomAndDebbugingHeaders

`
java.lang.AssertionError: 
Expected :header-1-value; header-1-value2;header-1-value3
Actual   :null
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at org.junit.Assert.assertEquals(Assert.java:146)
	at org.carapaceproxy.server.mapper.BasicStandardEndpointMapperTest.testCustomAndDebbugingHeaders(BasicStandardEndpointMapperTest.java:599)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.github.tomakehurst.wiremock.junit.WireMockRule$1.evaluate(WireMockRule.java:79)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
`

--- a/carapace-server/src/main/java/org/carapaceproxy/core/ProxyRequestsManager.java
+++ b/carapace-server/src/main/java/org/carapaceproxy/core/ProxyRequestsManager.java
@@ -88,11 +88,8 @@ public class ProxyRequestsManager {
     private final Map<String, HttpClient> forwardersPool = new ConcurrentHashMap<>(); // endpoint_connectionpool -> forwarder to use
     private final ConnectionsManager connectionsManager = new ConnectionsManager();
 
-    private RuntimeServerConfiguration currentConfiguration;
-
     public ProxyRequestsManager(HttpProxyServer parent) {
         this.parent = parent;
-        this.currentConfiguration = parent.getCurrentConfiguration();
     }
 
     public EndpointStats getEndpointStats(EndpointKey key) {
@@ -349,7 +346,7 @@ public class ProxyRequestsManager {
             return HttpClient.create(connectionProvider)
                     .host(endpointHost)
                     .port(endpointPort)
-                    .followRedirect(false) // clients has to request the redirect, not the proxy
+                    .followRedirect(false) // client has to request the redirect, not the proxy
                     .compress(parent.getCurrentConfiguration().isRequestCompressionEnabled())
                     .responseTimeout(Duration.ofMillis(connectionConfig.getStuckRequestTimeout()))
                     .option(ChannelOption.CONNECT_TIMEOUT_MILLIS, connectionConfig.getConnectTimeout())
@@ -363,7 +360,7 @@ public class ProxyRequestsManager {
                     .option(Epoll.isAvailable()
                             ? EpollChannelOption.TCP_KEEPCNT
                             : NioChannelOption.of(ExtendedSocketOptions.TCP_KEEPCOUNT), connectionConfig.getKeepaliveCount())
-                    .httpResponseDecoder(option -> option.maxHeaderSize(currentConfiguration.getMaxHeaderSize()))
+                    .httpResponseDecoder(option -> option.maxHeaderSize(parent.getCurrentConfiguration().getMaxHeaderSize()))
                     .doOnRequest((req, conn) -> {
                         if (CarapaceLogger.isLoggingDebugEnabled()) {
                             CarapaceLogger.debug("Start sending request for "
@@ -532,7 +529,7 @@ public class ProxyRequestsManager {
             headers.add(HttpHeaderNames.EXPIRES, HttpUtils.formatDateHeader(new java.util.Date(content.getExpiresTs())));
             request.setResponseHeaders(headers);
             addCustomResponseHeaders(request, request.getAction().customHeaders);
-            //If the request is http 1.0 we make sure to send without chunked
+            // If the request is http 1.0, we make sure to send without chunked
             if (parent.getCurrentConfiguration().isHttp10BackwardCompatibilityEnabled() &&
                     request.getRequest().version() == HttpVersion.HTTP_1_0) {
                 return request.sendResponseData(Mono.from(ByteBufFlux.fromIterable(content.getChunks())));
@@ -544,7 +541,7 @@ public class ProxyRequestsManager {
         }
     }
 
-    public class ConnectionsManager implements AutoCloseable, Function<ProxyRequest, Map.Entry<ConnectionPoolConfiguration, ConnectionProvider>> {
+    public static class ConnectionsManager implements AutoCloseable, Function<ProxyRequest, Map.Entry<ConnectionPoolConfiguration, ConnectionProvider>> {
 
         private final Map<ConnectionPoolConfiguration, ConnectionProvider> connectionPools = new ConcurrentHashMap<>();
         private volatile Map.Entry<ConnectionPoolConfiguration, ConnectionProvider> defaultConnectionPool;

--- a/carapace-server/src/test/java/org/carapaceproxy/backends/ChunkedEncodingResponseTest.java
+++ b/carapace-server/src/test/java/org/carapaceproxy/backends/ChunkedEncodingResponseTest.java
@@ -20,7 +20,9 @@
 package org.carapaceproxy.backends;
 
 import com.github.tomakehurst.wiremock.junit.WireMockRule;
-import io.netty.handler.codec.http.HttpHeaderNames;
+import java.util.Objects;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
 import org.apache.http.HttpResponse;
 import org.apache.http.HttpVersion;
 import org.apache.http.client.methods.HttpGet;
@@ -34,13 +36,17 @@ import org.carapaceproxy.utils.TestEndpointMapper;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static io.netty.handler.codec.http.HttpHeaderNames.CONTENT_LENGTH;
+import static io.netty.handler.codec.http.HttpHeaderNames.TRANSFER_ENCODING;
 import static org.junit.Assert.*;
 
 /**
  * @author enrico.olivelli
  */
+@RunWith(JUnitParamsRunner.class)
 public class ChunkedEncodingResponseTest {
 
     @Rule
@@ -50,7 +56,7 @@ public class ChunkedEncodingResponseTest {
     public TemporaryFolder tmpDir = new TemporaryFolder();
 
     @Test
-    public void testSimpleChunckedResponseNoCache() throws Exception {
+    public void testSimpleChunkedResponseNoCache() throws Exception {
         wireMockRule.stubFor(
                 get(urlEqualTo("/index.html")).
                         willReturn(aResponse()
@@ -62,7 +68,7 @@ public class ChunkedEncodingResponseTest {
         EndpointKey key = new EndpointKey("localhost", wireMockRule.port());
 
         CarapaceLogger.setLoggingDebugEnabled(true);
-        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder());) {
+        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder())) {
             server.start();
             int port = server.getLocalPort();
 
@@ -70,32 +76,41 @@ public class ChunkedEncodingResponseTest {
                 RawHttpClient.HttpResponse resp = client.executeRequest("GET /index.html HTTP/1.1\r\nHost: localhost\r\n\r\n");
                 String s = resp.toString();
                 System.out.println("s:" + s);
-                assertTrue(s.contains("12\r\n"
-                        + "it <b>works</b> !!\r\n"
-                        + "0\r\n\r\n"));
-                assertTrue(resp.getBodyString().equals("it <b>works</b> !!"));
+                assertTrue(s.contains("""
+                        12\r
+                        it <b>works</b> !!\r
+                        0\r
+                        \r
+                        """));
+                assertEquals("it <b>works</b> !!", resp.getBodyString());
 
                 resp = client.executeRequest("GET /index.html HTTP/1.1\r\nHost: localhost\r\n\r\n");
                 System.out.println("s:" + resp);
-                assertTrue(resp.toString().contains("12\r\n"
-                        + "it <b>works</b> !!\r\n"
-                        + "0\r\n\r\n"));
-                assertTrue(resp.getBodyString().equals("it <b>works</b> !!"));
+                assertTrue(resp.toString().contains("""
+                        12\r
+                        it <b>works</b> !!\r
+                        0\r
+                        \r
+                        """));
+                assertEquals("it <b>works</b> !!", resp.getBodyString());
             }
 
             try (RawHttpClient client = new RawHttpClient("localhost", port)) {
                 String s = client.executeRequest("GET /index.html HTTP/1.1\r\nHost: localhost\r\n\r\n").toString();
                 System.out.println("s:" + s);
-                assertTrue(s.contains("12\r\n"
-                        + "it <b>works</b> !!\r\n"
-                        + "0\r\n\r\n"));
+                assertTrue(s.contains("""
+                        12\r
+                        it <b>works</b> !!\r
+                        0\r
+                        \r
+                        """));
             }
             assertEquals(0, server.getCache().getCacheSize());
         }
     }
 
     @Test
-    public void testSimpleChunckedResponseWithCache() throws Exception {
+    public void testSimpleChunkedResponseWithCache() throws Exception {
         wireMockRule.stubFor(
                 get(urlEqualTo("/index.html")).
                         willReturn(aResponse()
@@ -106,36 +121,46 @@ public class ChunkedEncodingResponseTest {
         TestEndpointMapper mapper = new TestEndpointMapper("localhost", wireMockRule.port(), true);
         EndpointKey key = new EndpointKey("localhost", wireMockRule.port());
 
-        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder());) {
+        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder())) {
             server.start();
             int port = server.getLocalPort();
+            resetCache(server);
 
             try (RawHttpClient client = new RawHttpClient("localhost", port)) {
                 RawHttpClient.HttpResponse resp = client
                         .executeRequest("GET /index.html HTTP/1.1\r\nHost: localhost\r\n\r\n");
                 String s = resp.toString();
                 System.out.println("s:" + s);
-                assertTrue(s.contains("12\r\n"
-                        + "it <b>works</b> !!\r\n"
-                        + "0\r\n\r\n"));
-                assertTrue(resp.getBodyString().equals("it <b>works</b> !!"));
+                assertTrue(s.contains("""
+                        12\r
+                        it <b>works</b> !!\r
+                        0\r
+                        \r
+                        """));
+                assertEquals("it <b>works</b> !!", resp.getBodyString());
 
                 resp = client
                         .executeRequest("GET /index.html HTTP/1.1\r\nHost: localhost\r\n\r\n");
                 System.out.println("s:" + resp);
-                assertTrue(resp.toString().contains("12\r\n"
-                        + "it <b>works</b> !!\r\n"
-                        + "0\r\n\r\n"));
-                assertTrue(resp.getBodyString().equals("it <b>works</b> !!"));
+                assertTrue(resp.toString().contains("""
+                        12\r
+                        it <b>works</b> !!\r
+                        0\r
+                        \r
+                        """));
+                assertEquals("it <b>works</b> !!", resp.getBodyString());
             }
 
             try (RawHttpClient client = new RawHttpClient("localhost", port)) {
                 String s = client
                         .executeRequest("GET /index.html HTTP/1.1\r\nHost: localhost\r\n\r\n").toString();
                 System.out.println("s:" + s);
-                assertTrue(s.contains("12\r\n"
-                        + "it <b>works</b> !!\r\n"
-                        + "0\r\n\r\n"));
+                assertTrue(s.contains("""
+                        12\r
+                        it <b>works</b> !!\r
+                        0\r
+                        \r
+                        """));
             }
             assertEquals(1, server.getCache().getCacheSize());
             assertEquals(2, server.getCache().getStats().getHits());
@@ -145,7 +170,8 @@ public class ChunkedEncodingResponseTest {
 
 
     @Test
-    public void testChunkedHttp10() throws Exception {
+    @Parameters(method = "parametersForChunkedHttp10Test")
+    public void testChunkedHttp(final HttpVersion httpVersion, final boolean inCache) throws Exception {
         wireMockRule.stubFor(
                 get(urlEqualTo("/index.html")).
                         willReturn(aResponse()
@@ -156,63 +182,58 @@ public class ChunkedEncodingResponseTest {
         TestEndpointMapper mapper = new TestEndpointMapper("localhost", wireMockRule.port(), true);
         EndpointKey key = new EndpointKey("localhost", wireMockRule.port());
 
-        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder());) {
+        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder())) {
             server.start();
             int port = server.getLocalPort();
-
             server.getCurrentConfiguration().setHttp10BackwardCompatibilityEnabled(true);
+            resetCache(server);
 
-            CloseableHttpClient httpclient = HttpClients.createDefault();
-            HttpGet request = new HttpGet("http://localhost:" + port + "/index.html");
-            request.setProtocolVersion(HttpVersion.HTTP_1_1);
-            HttpResponse httpresponse = httpclient.execute(request);
+            try (CloseableHttpClient client = HttpClients.createDefault()) {
+                if (inCache) {
+                    HttpGet request = new HttpGet("http://localhost:" + port + "/index.html");
+                    request.setProtocolVersion(httpVersion);
+                    client.execute(request);
+                }
 
-            assertEquals("chunked",
-                    httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.TRANSFER_ENCODING)).getValue());
-            assertNull(httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.CONTENT_LENGTH)));
+                HttpGet request = new HttpGet("http://localhost:" + port + "/index.html");
+                request.setProtocolVersion(httpVersion);
+                HttpResponse response = client.execute(request);
 
-            server.getCache().getStats().resetCacheMetrics();
-            server.getCache().clear();
-            assertEquals(0, server.getCache().getCacheSize());
-            assertEquals(0, server.getCache().getStats().getHits());
-
-            request.setProtocolVersion(HttpVersion.HTTP_1_0);
-            httpresponse = httpclient.execute(request);
-
-            assertNull(httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.TRANSFER_ENCODING)));
-            assertNotNull(httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.CONTENT_LENGTH)));
-            assertNull(httpresponse.getFirstHeader("X-cached"));
-
-            //File must be in cache
-            assertEquals(1, server.getCache().getCacheSize());
-
-            CloseableHttpClient httpclient2 = HttpClients.createDefault();
-
-            request.setProtocolVersion(HttpVersion.HTTP_1_1);
-            httpresponse = httpclient2.execute(request);
-
-            //Response must come from cache
-            assertEquals(1, server.getCache().getStats().getHits());
-
-            //Http 1.1 chunked encoding
-            assertEquals("chunked",
-                    httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.TRANSFER_ENCODING)).getValue());
-            assertNull(httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.CONTENT_LENGTH)));
-            assertNotNull(httpresponse.getFirstHeader("X-cached"));
-
-
-            request.setProtocolVersion(HttpVersion.HTTP_1_0);
-            httpresponse = httpclient2.execute(request);
-
-            //Response must come from cache
-            assertEquals(2, server.getCache().getStats().getHits());
-            //Http 1.0 no chunked encoding
-            assertNull(httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.TRANSFER_ENCODING)));
-            assertNotNull(httpresponse.getFirstHeader(String.valueOf(HttpHeaderNames.CONTENT_LENGTH)));
-            assertNotNull(httpresponse.getFirstHeader("X-cached"));
+                if (Objects.equals(httpVersion, HttpVersion.HTTP_1_0)) {
+                    assertNull(response.getFirstHeader(TRANSFER_ENCODING.toString()));
+                    assertNotNull(response.getFirstHeader(CONTENT_LENGTH.toString()));
+                } else {
+                    assertEquals("chunked", response.getFirstHeader(TRANSFER_ENCODING.toString()).getValue());
+                    assertNull(response.getFirstHeader(CONTENT_LENGTH.toString()));
+                }
+                if (inCache) {
+                    assertNotNull(response.getFirstHeader("X-cached"));
+                    assertEquals(1, server.getCache().getStats().getHits());
+                } else {
+                    assertNull(response.getFirstHeader("X-cached"));
+                    assertEquals(0, server.getCache().getStats().getHits());
+                }
+            }
 
         }
     }
 
+    private static void resetCache(final HttpProxyServer server) {
+        server.getCache().reloadConfiguration(server.getCurrentConfiguration());
+        server.getCache().getStats().resetCacheMetrics();
+        server.getCache().clear();
+        assertEquals(0, server.getCache().getCacheSize());
+        assertEquals(0, server.getCache().getStats().getHits());
+        assertEquals(0, server.getCache().getStats().getMisses());
+    }
+
+    public static Object[] parametersForChunkedHttp10Test() {
+        return new Object[] {
+                new Object[] { HttpVersion.HTTP_1_0, false },
+                new Object[] { HttpVersion.HTTP_1_0, true },
+                new Object[] { HttpVersion.HTTP_1_1, false },
+                new Object[] { HttpVersion.HTTP_1_1, true },
+        };
+    }
 
 }

--- a/carapace-server/src/test/java/org/carapaceproxy/server/mapper/BasicStandardEndpointMapperTest.java
+++ b/carapace-server/src/test/java/org/carapaceproxy/server/mapper/BasicStandardEndpointMapperTest.java
@@ -487,7 +487,7 @@ public class BasicStandardEndpointMapperTest {
     }
 
     @Test
-    public void testCustomAndDebbugingHeaders() throws Exception {
+    public void testCustomAndDebuggingHeaders() throws Exception {
         stubFor(get(urlEqualTo("/index.html"))
                 .willReturn(aResponse()
                         .withStatus(200)

--- a/pom.xml
+++ b/pom.xml
@@ -141,7 +141,7 @@
         <!-- test dependecies -->
         <libs.junit>4.13.2</libs.junit>
         <libs.junitparams>1.1.1</libs.junitparams>
-        <libs.wiremock>2.32.0</libs.wiremock>
+        <libs.wiremock>2.35.0</libs.wiremock>
         <libs.powermock>2.0.9</libs.powermock>
         <libs.mockito>3.12.4</libs.mockito>
         <libs.hamcrest>2.2</libs.hamcrest>

